hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/json/pharmanet/ProcessV2ToPharmaNetJson.java

```diff
@@ -35,9 +34,9 @@ public void process(Exchange exchange) throws Exception {
 			throw new IllegalArgumentException("v2Message can't be null or empty");
```

Comment:
Do we need to throw CusotmHNSException? as CusotmHNSExceptionis handled in Route.java to  generate a response with http code as 403?

Reply:
Added use of CusotmHNSException